tests/system/migrations/conftest.py

```diff
@@ -144,15 +152,18 @@ def _read_model(fqid, position=None):
 @pytest.fixture()
 def assert_model(read_model):
     def _assert_model(fqid, expected, position=None):
-        if position is None:
-            assert read_model(fqid) == expected
-
-            # get max position
-            read_database: ReadDatabase = injector.get(ReadDatabase)
-            with read_database.get_context():
-                position = read_database.get_max_position()
-
-        # build model and check
-        assert read_model(fqid, position=position) == expected
+        try:
+            if position is None:
+                assert read_model(fqid) == expected
```

Comment:
Sorry, but what is "expected"?

Reply:
The expected value of the model, see a few lines higher. What do you wish to be changed here? I could introduce a type annotation if that's what you mean, but we don't have any of them in any other migration tests. I could also add an explaining comment, but I think the name and usage is verbose enough to be self-explanatory.